src/pages/Login.js

```diff
@@ -27,26 +33,57 @@ export const ButtonContainer = styled.div`
 // TODO: authentication
 export default () => {
   const theme = useContext(ThemeContext)
+  const { setUser } = useAuth()
+  const [, setLocation] = useLocation()
+  const [showError, setShowError] = useState(false)
+
+  const showErrorMessage = error => {
+    if (!error) return
+    setShowError(true)
+    setTimeout(() => {
+      setShowError(false)
+    }, 5000)
```

Comment:
Can we increase the timeout here?
```suggestion
    }, 10000)
```